Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster capable of sustained cold burns which can be applied to a CubeSat to achieve the reuqired
+velocity to reenter Earth's atmosphere. This is a system property, however using the thrust over time curves reasonable estimations can be made to say whether or not it is possible.
+A target specific impulse is 50s. This can be determined from testing by integrating the thrust over time curve. See the equation 1, below, to see why this is possible.
+
+\begin{equation} \label{specific impulse}
+  I_t    = \int_{t_i}^{t_f} F_{thrust} dt
+\end{equation}
+
+\begin{equation} \label{specific impulse}
+  I_s = \frac{I_t}{m_p g_0}
+\end{equation}
+
+Where $m_p$ is the propellant mass and $g_0$ is standard gravity, 9.81 $ms^{-2}$.
+As seen above, calculating the specific impulse is
+
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/cold-gas-thruster.png}
+  \caption{This is the Solidworks Assembly of the current design of the SPEX cold gas thruster. The piping interface to the nozzle is omitted for clarity.}
+\label{fig:lifecycle}
+\end{figure}
+
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data.
+However, from previous testing noise is not a problem but error is slightly. The error in the last tests maxed at 0.05 Newtons, which is no terribly significant but still should be accounted.
+Vacuum condition availablity is low at RIT, so the team will need to account for ambient conditions. This primarily includes ambient pressure for achieveing maximum thrust, but there are other
+nuisances. The use of this cold gas thruster will give the team experience in the fabrication of propulsion systems and that can be used for designing more complex systems. These systems could
+ include a CubeSat-sized propulsion system.
+
+
+
+% \section{Secondary Objectives}
+% \label{sec:secondary-obj}
+% Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+This project would be benefical to SPEX because of the pathway it creates. After its completion, SPEX will have a cold gas thruster that
+be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy, and since
```

Comment:
I don't think that is something to go into in this section. It does not have to do with the design, it is about how the project benefits SPEX. I don't think this is place for design analysis.